src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/KeywordControllerTest.php

```diff
@@ -104,7 +104,7 @@ public function testGet()
     {
         $keyword = $this->testPost('keyword1', 'de', $this->category1->getId());
 
-        $this->client->request(
+        static::jsonRequest($this->client,
```

Comment:
The parameter should be in a separate line, like in all other places in this PR as well.